spring-cloud-dataflow-shell-core/src/main/resources/usage.txt

```diff
@@ -2,7 +2,7 @@ Data Flow Options:
   --dataflow.uri=<uri>                  Address of the Data Flow Server [default: http://localhost:9393].
   --dataflow.username=<USER>            Username of the Data Flow Server [no default].
   --dataflow.password=<PASSWORD>        Password of the Data Flow Server [no default].
-  --spring.shell.historySize=<SIZE>     Default size of the shell log file [default: 3000]
+  --dataflow.skip-ssl-validation=true   Accept any SSL certificate (even self-signed) [default: no].
```

Comment:
defaults are specified in square brackets, not inline. So should read `=true|false` or something similar